viewer-config-persistence/src/main/java/nl/b3p/viewer/util/databaseupdate/DatabaseSynchronizer.java

```diff
@@ -174,6 +174,11 @@
         
         // 5.7.3
         updates.put("39", new UpdateElement(Collections.singletonList("add_geoservice_geofence_header.sql"), String.class, true));
+
+        //5.6.4
+
+        updates.put("40", new UpdateElement (Collections.singletonList("selectedcontentcaches_dirty.sql"), String.class));
```

Comment:
this file is missing from this PR

Reply:
It's an older, already existing file which I reused